uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/OpenTabActionUiTest.scala

```diff
@@ -12,11 +11,21 @@ import com.virtuslab.ideprobe.{ IntegrationTestSuite, IntelliJFixture }
 import org.junit.{ Assert, Test }
 
 class OpenTabActionUiTest extends BaseGitRepositoryBackedTest with IntegrationTestSuite {
-  val currentDir: File = new File(System.getProperty("user.dir"))
-  val rootDir: File = currentDir.getParentFile
-  val distDir: File = rootDir.toPath.resolve("build").resolve("distributions").toFile
-  val latestFileInDistDir: File = distDir.listFiles().maxBy(_.lastModified)
-  val pluginUri: URI = new URI(latestFileInDistDir.getAbsolutePath)
+
+  implicit class ProjectDirOps(projectDir: File) {
+
+    /** @return the latest file in <projectDir>/build/distributions */
+    def pluginFile: File = projectDir.toPath
+      .resolve("build").resolve("distributions").toFile
+      .listFiles()
+      .maxBy(_.lastModified)
+  }
+
+  // `user.dir` property resolves to the current subproject path in tests executed by Gradle `test` task.
+  val currentSubprojectDir: File = new File(System.getProperty("user.dir"))
+  val uiTestActionsPlugin: File = currentSubprojectDir.pluginFile
```

Comment:
In fact there's little magic here, this just desugars into `(new ProjectDirOps(currentSubprojectDir)).pluginFile`